src/broker/index.js

```diff
@@ -703,6 +703,77 @@ module.exports = class Broker {
     return await this.connection.send(deleteGroups(groupIds))
   }
 
+  /**
+   * @public
+   * @param {Array} ACL e.g:
+   *                 [
+   *                   {
+   *                     resourceType: ,
+   *                     resourceName: 'resource-name',
+   *                     resourcePatternType: ,
+   *                     principal: 'principal-name',
+   *                     host: 'hostname',
+   *                     operation: ,
+   *                     permissionType:
+   *                   }
+   *                 ]
+   * @returns {Promise<void>}
+   */
+  async createAcls({ acl }) {
+    const createAcls = this.lookupRequest(apiKeys.CreateAcls, requests.CreateAcls)
+    return await this.connection.send(createAcls({ creations: acl }))
+  }
+
+  /**
+   * @public
+   * @param {number} resourceType The type of resource
```

Comment:
Rather than using primitive types here, declare an enum type in the corresponding protocol file and reference that here. For example:

```js
@param {AclOperation} The type of ACL operation
```